diff -Naurp old/config.guess new/config.guess
--- old/config.guess	2008-10-28 01:47:05.000000000 +0530
+++ new/config.guess	2014-10-13 10:00:29.328163608 +0530
@@ -894,6 +894,9 @@ EOF
     ppc64:Linux:*:*)
 	echo powerpc64-unknown-linux-gnu
 	exit 0 ;;
+    ppc64le:Linux:*:*)
+	echo powerpc64le-unknown-linux-gnu
+	exit 0 ;;
     alpha:Linux:*:*)
 	case `sed -n '/^cpu model/s/^.*: \(.*\)/\1/p' < /proc/cpuinfo` in
 	  EV5)   UNAME_MACHINE=alphaev5 ;;
